torch/_dynamo/symbolic_convert.py

```diff
@@ -1017,7 +1037,8 @@ def UNPACK_SEQUENCE(self, inst):
         seq = self.pop()
         options = VariableTracker.propagate([seq])
         if isinstance(seq, BaseListVariable):
-            assert len(seq.items) == inst.argval
+            if len(seq.items) != inst.argval:
+                unimplemented(f"UNPACK_SEQUENCE {seq}")
```

Comment:
Tell me more? Assert looks pretty legit to me

Reply:
Yea, we dont need this.